database-plugins/docs/Database-batchsource.md

```diff
@@ -16,58 +16,58 @@ a TimePartitionedFileSet.
 
 Properties
 ----------
-**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+**Reference Name:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
```

Comment:
Name used to uniquely identify this source for lineage, annotating metadata, etc.